### DISCUSSION TITLE
How to tweak the branding using the docker version?

### DISCUSSION THREAD

**Discussion by JobaDiniz** [structurizr/structurizr.github.io#13] [Q&A]
I don't see the documentation to tweak the branding in the docker version. The doc is empty: https://docs.structurizr.com/ui/branding

**Comment by simonbrowndotje** [comment]
It looks like that page didn't get ported - I will add it. In the meantime, you can use the `branding` DSL keyword -> https://docs.structurizr.com/dsl/language#branding

**Reply by JobaDiniz** [reply]
where do I add that?
I got this error

```
workspace.dsl: Unexpected tokens (expected: name, description, properties, !docs, !decisions, !identifiers, !impliedRelationships, model, views, configuration) at line 5 of /usr/local/structurizr/workspace.dsl: branding {
```

**Reply by simonbrowndotje** [reply]
`branding` can be used inside the `views` block -> https://docs.structurizr.com/dsl/language#views

**Comment by simonbrowndotje** [comment]
The branding pages have now been ported:

- https://docs.structurizr.com/ui/diagrams/branding
- https://docs.structurizr.com/ui/documentation/branding
